e2e/nodedrain/nodedrain.go

```diff
@@ -314,7 +316,7 @@ func (tc *NodeDrainE2ETest) TestNodeDrainForce(f *framework.F) {
 				}
 			}
 			return false
-		}, &e2e.WaitConfig{Interval: time.Millisecond * 100, Retries: 200},
+		}, &e2e.WaitConfig{Interval: time.Second, Retries: 40},
```

Comment:
node-drain failures are suspecious.  `TestNodeDrainDeadline ` flakiness seems to be due to timing effects, but `TestNodeDrainForce` may be legitimate.  In some of my build samples, TestNodeDrainForce failed, because the client did wait the full 2m minutes before killing the node.  Sample build https://app.circleci.com/pipelines/github/hashicorp/nomad-e2e/58/workflows/410109f1-2c7c-4cae-ae98-41b20358776e/jobs/286/steps